db/migrate/20151102211125_add_slug_to_bit_core_tool.rb

```diff
@@ -0,0 +1,5 @@
+class AddSlugToBitCoreTool < ActiveRecord::Migration
+  def change
+    add_column :bit_core_tools, :slug, :string
```

Comment:
Do you think it would be fine to add a default of an empty string?  Or should the slug be the current title of the tool; however, this seems redundant?  In addition, the `title` of each tool isn't currently unique.  With your recent point, should I update the validation on the `title` attribute so it is unique for each arm as well?